app/views/support/WeeklyPromotion.scala

```diff
@@ -72,43 +51,109 @@ object WeeklyPromotion {
       val countries = promotionCountries intersect UK
       val includesUKDomestic = (countries intersect UKdomestic).nonEmpty
       val includesUKOverseas = (countries intersect UKoverseas).nonEmpty
+
       if(includesUKDomestic && includesUKOverseas){
-        Set(all)
+        all
       } else if (includesUKDomestic) {
-        Set(domestic)
-      } else if (includesUKOverseas) {
-        Set(overseas)
+        domestic
       } else {
-        Set()
+        overseas
       }
     }
-    val USregion = Seq(DiscountedRegion(
+    val USregion = DiscountedRegion(
       title = "United States",
       description = "Includes Alaska and Hawaii",
       discountedPlans = plansForPromotion(promotion, promoCode, WeeklyPicker.product(Country.US), Currency.USD)
-    ))
-    val AUSregion = Seq(DiscountedRegion(
+    )
+    val AUSregion = DiscountedRegion(
       title = "Australia",
       description = "Posted to you by air mail",
       discountedPlans = plansForPromotion(promotion, promoCode, WeeklyPicker.product(Country.Australia), Currency.AUD)
-    ))
-    val NZregion = Seq(DiscountedRegion(
+    )
+    val NZregion = DiscountedRegion(
       title = "New Zealand",
       description = "Posted to you by air mail",
       discountedPlans = plansForPromotion(promotion, promoCode, WeeklyPicker.product(Country.NewZealand), Currency.NZD)
-    ))
-    val CAregion = Seq(DiscountedRegion(
+    )
+    val CAregion = DiscountedRegion(
       title = "Canada",
       description = "Posted to you by air mail",
       discountedPlans = plansForPromotion(promotion, promoCode, WeeklyPicker.product(Country.Canada), Currency.CAD)
-    ))
-    val EUregion = Seq(DiscountedRegion(
+    )
+    val EUregion = DiscountedRegion(
       title = "Europe",
       description = "Posted to you by air mail",
       discountedPlans = plansForPromotion(promotion, promoCode, WeeklyPicker.productForCountryGroup(CountryGroup.Europe), Currency.EUR)
+    )
+
+    Map(
+      CountryGroup.UK -> UKregion,
+      CountryGroup.US -> USregion,
+      CountryGroup.Europe -> EUregion,
+      CountryGroup.Australia -> AUSregion,
+      CountryGroup.NewZealand -> NZregion,
+      CountryGroup.Canada -> CAregion
+    )
+  }
+
+  //If the request country belongs to a domestic country group, we still still display that country at the top of the list
+  //However, if it corresponds to a region that is always displayed then we need to ensure we don't show it twice
+  //e.g. if the request country is the US, then it is the prioritised country and we need to not show it again in the
+  //list of domestic countries. A rest of world country should always be at the top, and then all the domestic country groups after.
+  def domesticCountryGroupsToDisplay(requestCountry: Country): List[CountryGroup] = {
```

Comment:
One possible alternative to this de-duping would be to add a _prioritised_ field to the DiscountedRegion case class.

We'd then need to define a custom ordering based on this value, but once we had that it would allow us to sort the full list of discounted regions.

Reply:
Actually we could probably just use sortBy rather than bothering with an ordering...

Reply:
Hmm, good idea. Let me give that a try.